README.md

```diff
@@ -17,28 +17,27 @@ suitable learning resources for those.
 ## Download and Run Courses
 
 The most flexible learning method is to download this repository and run it in a
-Python environment with the correct dependencies.  
-
+Python environment with the correct dependencies.
 A suitable environment can be produced with
-[conda](https://docs.conda.io/en/latest/miniconda.html")
-(only 'miniconda' is needed).  
-To create the environment, use the command: 
-`conda create -n testenv iris iris-sample-data jupyter`.  
+[conda](https://docs.conda.io/en/latest/miniconda.html)
+(only 'miniconda' is needed) :
+
+  * To create the environment, use the command :  
+    `$ conda create -n testenv iris iris-sample-data jupyter`
 
-To run the courses, navigate to the repository 'course_content' folder
-and run: `jupyter notebook`.
+  * To run the courses, navigate to the repository 'course_content' folder
+    and run :  
+    `$ jupyter notebook`
 
 ## Run Courses in Binder
 
 It is also possible to run any of these courses in a cloud instance
 with [Binder](https://mybinder.org/),
-simply by following the links provided below.  
-
-This is quick to try, and highly convenient,
-**however** you must be aware that this runs in a transient session,
-so all your edits will be lost when that expires
-(usually some 10s of minutes after the last interaction).  
-
+simply by following the links provided below.
+This is quick to try, and highly convenient.
+**However** _you must be aware that this runs in a transient session,_
+_so all your edits will be lost when that expires_
+_(usually some 10s of minutes after the last interaction)._
```

Comment:
I don't like the sentence starting with "However..." being lost in the paragraph. I'd prefer a warning like that being on its on line.